contribute-to-tidb/review-a-pr.md

```diff
@@ -7,46 +7,163 @@ This guide is for all contributors that want to help with reviewing code contrib
 * We make sure to keep a high code quality in TiDB.
 * We avoid situations where contributors and reviewers spend a lot of time refining a contribution that gets rejected later.
 
-Every review needs to check the following six aspects. We encourage to check these aspects in order, to avoid spending time on detailed code quality reviews when formal requirements are not met or there is no consensus in the community to accept the change.
 
-## 1. Is the Contribution Well-Described?
+## 1. Things to know before the code review
 
-Check whether the contribution is sufficiently well-described to support a good review. Trivial changes and fixes do not need a long description. If the implementation is exactly according to a prior discussion on an issue or the [TiDB Internals forum](https://internals.tidb.io/), only a short reference to that discussion is needed. If the implementation is different from the agreed approach in the consensus discussion, a detailed description of the implementation is required for any further review of the contribution.
+Why is code review important?
 
-Any pull request that changes functionality or behavior needs to describe the big picture of these changes, so that reviews know what to look for \(and don't have to dig through the code to hopefully understand what the change does\).
+* Erase bugs early
+* Make the project maintainable in the long run
 
-A contribution is well-described if the following questions 2, 3, and 4 can be answered without looking at the code.
+Some high level guidelines about code review in PingCAP (and mostly applies the TiDB community also):
```

Comment:
```suggestion
Some high level guidelines about code review in the TiDB community:
```

Yep. It could be guidelines in PingCAP. But they are also contributors in TiDB community and actually how TiDB community works. We don't have to emphasize "PingCAP" here.

Reply:
Okay... writing much more PRs than reviewing is common in our company, especially for the new employees.
For the community contributors, they don't have such strong feeling.

Reply:
At least for this sentence, it emphasizes the importance of review itself. I don't feel an impact on "writing much more PRs than reviewing" among these words.